jekyll/_cci2/github-app-trigger-options.adoc

```diff
@@ -0,0 +1,83 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= GitHub App trigger options
+:page-layout: classic-docs
+:page-liquid:
+:page-description: A guide to the options available for triggering GitHub App pipelines
+:icons: font
+:experimental:
+
+When you create a trigger for a GitHub App pipeline, you can choose from a set of trigger options that correspond to link:https://docs.github.com/en/webhooks/webhook-events-and-payloads[GitHub App events].
+
+== Introduction
+
+When you set up a project in CircleCI you will also set up your first pipeline and trigger. This first "default" trigger is an "all push" trigger, which corresponds to the link:https://docs.github.com/en/webhooks/webhook-events-and-payloads#push[GitHub App push event]. You can also set up additional triggers for your project during setup, or later in your menu:Project settings[Triggers].
```

Comment:
> When you set up a project in CircleCI
I think we should also think about OAuth customers here, and say something like
```suggestion
When you set up your first GitHub App pipeline, either through Project Settings or through the project creation flow, you will also set up your first trigger for your pipeline. Triggers define on what events and conditions your pipeline should run. 
When setting up a GitHub App trigger, you will be presented with a range of trigger event options. The default option is to trigger a pipeline on "all pushes", which corresponds to the link:https://docs.github.com/en/webhooks/webhook-events-and-payloads#push[GitHub App push event], but this can be changed to any of the alternative options in the dropdown. 
```

Reply:
I don't actually think this suggestion is good, but I wanted to make sure that classic org users don't start reading this and think "hold on it's not me, I'm not creating a new project"

Reply:
Also - I want to make it clear that during project onboarding they can choose a different option in the dropdown

Reply:
I'm going to add a screenshot of the menu in the create project flow and change the wording a bit